### PR TITLE
Enhancement/add symmetry wall bc

### PR DESCRIPTION
**Pull request by nnetake**
added symmetry wall bc and unit test for the same

### CODE REVIEW

**Comment by lorenzovecchietti** [comment]
@nnetake I removed one line from the unit test so to improve coverage and a made a few changes in the return of various bc functions to make them more homogeneous

File: pyaedt/icepak.py [hunk 1]
```diff
@@ -4643,3 +4655,48 @@ def assign_mass_flow_free_opening(
...
+        ----------
+
+        >>> oModule.AssignSymmetryWallBoundary
+        """
+        if not boundary_name:
+            boundary_name = generate_unique_name("StationaryWall")
+        if isinstance(geometry, str):
+            geometry = [geometry]
+        elif isinstance(geometry, int):
+            geometry = [geometry]
```
Comment (MaxJPRey) [inline pyaedt/icepak.py:4689]:
```suggestion
        if isinstance(geometry, str) or isinstance(geometry, int):
            geometry = [geometry]
```

Comment (nnetake) [reply pyaedt/icepak.py:4689]: Thanks for this suggestion!

File: pyaedt/icepak.py [hunk 2]
```diff
@@ -4643,3 +4667,49 @@ def assign_mass_flow_free_opening(
...
+    def assign_symmetry_wall(
+        self,
+        geometry,
+        boundary_name=None,
+    ):
+        """Assign symmetry wall boundary condition.
+
+        Parameters
+        ----------
+        geometry : str or int or list of str/int
```
Comment (PipKat) [inline pyaedt/icepak.py:4681]:
```suggestion
        geometry : str or int or list
```

**Review by maxcapodi78 — APPROVED** [review]
LGTM

**Pull request merged**